services.yaml

```diff
@@ -1884,6 +1886,130 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         404:
           $ref: "#/responses/NotFoundError"
+#System Metrics
+  /services/{env_id}/metrics/system:
+    get:
+      tags:
+        - System Metrics
+      summary: "Get Metadatas"
+      description: "Get a list of all existing JVM Metrics"
```

Comment:
"Get a list of all available system metrics"

Reply:
fixed